lib/unhangout-sockets.js

```diff
@@ -243,6 +320,30 @@ _.extend(UnhangoutSocketManager.prototype, events.EventEmitter.prototype, {
             }
         }, this));
 
+        mgr.on("participant-proposed", _.bind(function(socket, args) {
+            var event = this.getEvent(args.roomId);
+            
+            if (event && socket.user && socket.user.isAdminOf(event)) {
+                event.save({adminProposedSessions: false});
+                mgr.writeAck(socket, "participant-proposed");
+                event.logAnalytics({action: "participant-proposed", user: socket.user});
+            } else {
+                mgr.writeErr(socket, "participant-proposed");
+            }
+        }, this));
+
```

Comment:
Corresponding changes are indicated in this commit  https://github.com/drewww/unhangout/commit/358eb7654ab2c6ac2fd489d32026687b3afa31df